pyscriptjs/src/interpreter.ts

```diff
@@ -39,20 +39,39 @@ const loadPackage = async function (package_name: string[] | string, runtime: an
 
 const loadFromFile = async function (s: string, runtime: any): Promise<any> {
     const filename = getLastPath(s);
-    await runtime.runPythonAsync(
-        `
-        from pyodide.http import pyfetch
-        from js import console
-        response = await pyfetch("` +
-            s +
-            `")
-        content = await response.bytes()
-        with open("` +
-            filename +
-            `", "wb") as f:
-            f.write(content)
-    `,
-    );
+    try {
+        await runtime.runPythonAsync(
+            `
+            from pyodide.http import pyfetch
+            from js import console
+
+            response = None;
```

Comment:
not sure I understand the need for this. Can you elaborate? (if it's needed, we should remove the `;` here)

Reply:
(See next comment)